src/ontology/mondo-edit.obo

```diff
@@ -225590,6 +225591,7 @@ xref: OMIM:278850 {source="MONDO:equivalentTo"}
 xref: Orphanet:393 {source="OMIM:278850"}
 xref: UMLS:C2749215 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:411414"}
 is_a: MONDO:0100249 {source="DOID:0111763", source="Orphanet:393/btnt"} ! 46,XX testicular disorder of sex development
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11204 {source="OMIM:278850"} ! SOX9
```

Comment:
We should not have this relationship to the gene because 
"(...) is caused by heterozygous duplication or triplication of a **68-kb regulatory region (XXSR) -584 to -516 kb upstream of the SOX9 gene** on chromosome 17q24."

However, one would not be able to see without reading the details. (ie this has nothing to do with this workflow).

I will manually remove it in a follow-up PR (I have something else to update). 

@twhetzel Could you please confirm: Should I add this Mondo/OMIM to the "excluded spreadsheet", so it will not be updated next time? What is the SOP? Thanks.

Reply:
Generally, there will be another file that can be updated by curators for gene associations that should not be added even though they otherwise meet the rules. Joe is working on this now.